spec/models/card_spec.rb

```diff
@@ -1,10 +1,18 @@
 require 'rails_helper'
 
-RSpec.describe Card do
+describe Card do
+
+  it { should belong_to :user }
 
   let(:card) { create(:card) }
+
   describe 'card has valid attributes' do
 
+    before { card.user = create(:user) }
+    it 'validates card has  user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.